nuxeo-features/nuxeo-admin-center/nuxeo-admin-center-jsf/src/main/java/org/nuxeo/connect/client/jsf/ConnectStatusActionBean.java

```diff
@@ -132,8 +132,8 @@ protected ConnectRegistrationService getService() {
     }
 
     @Factory(scope = ScopeType.APPLICATION, value = "registredConnectInstance")
```

Comment:
I don't want to change this one, because this bean `registredConnectInstance` is used in 26 places in 13 files, most of which are `*.xhtml` files: renaming them requires at least a manual functional test. Since this is not related to the ticket, I prefer to let somebody else to handle it.

But you might also ask why I renamed the other one:

```
return getService().isInstanceRegistred();
```

Because the Nuxeo Connect Client API `ConnectRegistrationService#isInstanceRegistred` has renamed to `ConnectRegistrationService#isInstanceRegistered` since commit https://github.com/nuxeo/nuxeo-connect/commit/7add6f98830e306376b8b0dcab3d5604836b5fdb without keeping the deprecated API, so I had to update it.